packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -370,4 +379,6 @@ class VideoPlayerOptions {
   /// Note: This option will be silently ignored in the web platform (there is
   /// currently no way to implement this feature in this platform).
   final bool mixWithOthers;
+
+  final bool bypassAudioSetup;
```

Comment:
This needs a comment explaining what it does, that it's iOS-only, and that it will be removed in the future (with a link to an issue).